design-library/source/_patterns/01-atoms/05-Search/_search.scss

```diff
@@ -0,0 +1,19 @@
+.search {
```

Comment:
I think you can rename the directory to use lowercase, e.g. `05-search`

Reply:
The directory would need to be a different number than `05-list-posted-on`. Please use a unique number